sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -515,18 +515,16 @@ export class JsonRpcProvider extends Provider {
     options?: SuiObjectDataOptions,
   ): Promise<SuiObjectResponse[]> {
     try {
-      const requests = objectIds.map((id) => {
+      objectIds.forEach((id) => {
```

Comment:
nice, could you also
1. remove a TODO comment (search for `multi-get`) in builder/Transaction.ts
2. resolve a todo comment in client_commands.rs(search for `multi-get`
3. throw an error if the objectIds/digests contain any duplicates? And also describe this requirement in the comments in provider.ts

Reply:
Added changes to all 3!

Reply:
Amazing!